File: src/main/csharp/JSONException.cs
/*
 * Copyright (c) 2016, Inversoft Inc., All Rights Reserved
 */

using System;

namespace Inversoft.Restify
{
  public class JSONException : Exception
  {
    public JSONException()
    {
    }

    public JSONException(Exception cause) : base(null, cause)
    {
    }
  }
}

File: src/main/java/com/inversoft/rest/JSONException.java
/*
 * Copyright (c) 2016-2019, Inversoft Inc., All Rights Reserved
 */
package com.inversoft.rest;

/**
 * @author Brian Pontarelli
 */
public class JSONException extends RuntimeException {
  public JSONException() {
    super();
  }

  public JSONException(Throwable cause) {
    super(cause);
  }

  public JSONException(String message, Throwable cause) {
    super(message, cause);
  }
}
